bin/pg_repack.c

```diff
@@ -287,6 +288,7 @@ static pgut_option options[] =
 	{ 'b', 'k', "no-superuser-check", &no_superuser_check },
 	{ 'l', 'C', "exclude-extension", &exclude_extension_list },
 	{ 'b', 'F', "error-on-invalid-index", &error_on_invalid_index },
+	{ 'i', 1, "switch-threshold", &switch_threshold },
```

Comment:
I'm confused by the meaning of such a constant here. What does the value 1 mean here? It is not a `required_argument from` getopt_long

Reply:
Basically it is a unique identifier of the option. It should be unique and other options shouldn't use it. Other option is to use `indexptr` of `getopt_long`, but pg_repack doesn't use it.
Options represented by alphabetic symbols (short option name) also shouldn't be same when you convert it to integer.
https://www.gnu.org/software/libc/manual/html_node/Getopt-Long-Options.html

pg_repack uses its own wrapper `struct pgut_option` around `getopt_long`. And second value is `sname`:
https://github.com/reorg/pg_repack/blob/944dbf7a49adfed7bd2ad467ae9431e8067442a4/bin/pgut/pgut-fe.h#L42

pg_repack copies `struct pgut_option` into `struct option` here:
https://github.com/reorg/pg_repack/blob/944dbf7a49adfed7bd2ad467ae9431e8067442a4/bin/pgut/pgut-fe.c#L656